db/migrate/20230907224544_update_user_for_devise.rb

```diff
@@ -0,0 +1,43 @@
+class UpdateUserForDevise < ActiveRecord::Migration[7.0]
+  def change
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: "email@mail.com"
+      t.string :encrypted_password, null: false
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
```

Comment:
- [ ] Please remove this migration and the ones below it because these are for a future project milestone `Authentication with devise`.

![image](https://github.com/Marcraphael12/rails-blog-app/assets/68796232/5873c84e-1542-4a4f-8f9c-3a9868f8fe5b)